nibabel/streamlines/tests/test_trk.py

```diff
@@ -1,10 +1,11 @@
 import os
+import sys
 import copy
 import unittest
 import numpy as np
 from os.path import join as pjoin
 
-from six import BytesIO
+from nibabel.externals.six import BytesIO
```

Comment:
Would you mind reverting this to `from six import BytesIO`?  We're trying to remove use of the `nibabel.externals.six` module.